Makefile

```diff
@@ -76,11 +76,11 @@ update:
 define build-version
 build-$1:
 ifeq ($(do_default),true)
-	$(DOCKER) build --pull -t $(REPO_NAME)/$(IMAGE_NAME):$(shell echo $1) $1
+	$(DOCKER) buildx build --platform linux/arm64,linux/amd64 -t $(REPO_NAME)/$(IMAGE_NAME):$(shell echo $1) --push $1
 endif
 ifeq ($(do_alpine),true)
 ifneq ("$(wildcard $1/alpine)","")
-	$(DOCKER) build --pull -t $(REPO_NAME)/$(IMAGE_NAME):$(shell echo $1)-alpine $1/alpine
+	$(DOCKER) buildx build --platform linux/arm64,linux/amd64 -t $(REPO_NAME)/$(IMAGE_NAME):$(shell echo $1)-alpine --push $1/alpine
```

Comment:
I believe the --push here will attempt to push the image up to dockerhub, but this this is the build target.  The push target is separate and defined separately so that the images can be built locally without pushing to dockerhub.  Building w/o pushing is the desired use-case for verifying PRs that are submitted but not merged.  Pushes to dockerhub will only happen on successful builds of a merge into the github repo.